packages/@react-spectrum/datepicker/docs/DatePicker.mdx

```diff
@@ -0,0 +1,312 @@
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/datepicker';
+import i18nDocs from 'docs:@internationalized/date';
+import {HeaderInfo, PropTable, PageDescription, TypeLink} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/datepicker/package.json';
+
+```jsx import
+import {DatePicker} from '@react-spectrum/datepicker';
+import {Flex} from '@react-spectrum/layout';
+```
+
+---
+category: Pickers
+keywords: []
+after_version: 3.0.0
+---
+
+# DatePicker
+
+<PageDescription>{docs.exports.DatePicker.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['DatePicker']}
+  sourceData={[]} />
+
+## Example
+
+```tsx example
+<DatePicker label="Event date" />
+```
+
+## Value
+
+A `DatePicker` displays a placeholder date by default. An initial, uncontrolled value can be provided to the `DatePicker` using the `defaultValue` prop. Alternatively, a controlled value can be provided using the `value` prop.
+
+Date values are provided using objects in the `@internationalized/date` package. This library handles correct international date manipulation across calendars, time zones, and other localization concerns. `DatePicker` supports values of the following types:
+
+* <TypeLink links={i18nDocs.links} type={i18nDocs.exports.CalendarDate} /> – a date without any time components. May be parsed from a string representation using the <TypeLink links={i18nDocs.links} type={i18nDocs.exports.parseDate} /> function. Use this type to represent dates where the time is not important, such as a birthday or an all day calendar event.
+* <TypeLink links={i18nDocs.links} type={i18nDocs.exports.CalendarDateTime} /> – a date with a time, but not in any specific time zone. May be parsed from a string representation using the <TypeLink links={i18nDocs.links} type={i18nDocs.exports.parseDateTime} /> function. Use this type to represent times that occur at the same local time regardless of the time zone, such as the time of New Years Eve fireworks. Most times are better stored as a `ZonedDateTime`.
+* <TypeLink links={i18nDocs.links} type={i18nDocs.exports.ZonedDateTime} /> – a date with a time in a specific time zone. May be parsed from a string representation using the <TypeLink links={i18nDocs.links} type={i18nDocs.exports.parseZonedDateTime} />, <TypeLink links={i18nDocs.links} type={i18nDocs.exports.parseAbsolute} />, or <TypeLink links={i18nDocs.links} type={i18nDocs.exports.parseAbsoluteToLocal} /> functions. Use this type to represent an exact moment in time at a particular location on Earth.
+
+```tsx example
+import {parseDate} from '@internationalized/date';
```

Comment:
How do we want to handle this? It's a separate library that we expect users to use, but it is also a dependency of DatePicker itself. Seems like a recipe for multiple versions getting included... But also, probably way too much to include in the RSP monopackage.

Reply:
There will be a separate section, and a link from here eventually